_pages/security/static-analysis.md

```diff
@@ -21,14 +21,19 @@ Service | Dependency analysis | Code security analysis
 
 #### Gemnasium
 
+Gemnasium is used to scan your code for possible security issues and provides alerts for new issues that come to light.
+
 ##### Setting up your account
 
-1. [Sign into Gemnasium via GitHub.](https://gemnasium.com/login)
+1. [Sign into Gemnasium using your GitHub account.](https://gemnasium.com/login)
 1. Go to [your settings page](https://gemnasium.com/settings) and change it to use your work email, if it isn't already.
 1. Ask in [#infrastructure](https://gsa-tts.slack.com/messages/infrastructure/) to be invited to the devops@gsa.gov account.
+1. This provides you with access to the common dashboard for our projects. Make sure that your project also has a backup with access to the dashboard.
```

Comment:
Consider changing the language around "backup" to make it clear it is a person backup. Maybe "Make sure your project has multiple people with access to the gemnasium dashboard."